compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```
@@ -4371,6 +4371,22 @@
                     "type": "ExpressionNode"
                 }
             ]
+        },
+        {
+            "name": "TupleMemberDescriptorNode",
```

```suggestion
            "name": "MemberTypeDescriptorNode",
```
@rdulmina WDYT? to be aligned with the spec terms.

---

+1